src/test/groovy/org/prebid/server/functional/model/response/auction/TraceOutcome.groovy

```diff
@@ -3,13 +3,12 @@ package org.prebid.server.functional.model.response.auction
 import com.fasterxml.jackson.databind.PropertyNamingStrategies
 import com.fasterxml.jackson.databind.annotation.JsonNaming
 import groovy.transform.ToString
-import org.prebid.server.functional.model.config.Stage
 
 @ToString(includeNames = true, ignoreNulls = true)
 @JsonNaming(PropertyNamingStrategies.LowerCaseStrategy)
 class TraceOutcome {
 
-    Stage entity
+    String entity
```

Comment:
Why not stage?

Reply:
Because BIDDER_REQUEST, RAW_BIDDER_RESPONSE, and PROCESSED_BIDDER_RESPONSE from beginning had bidder as entity and stage as a separate parameter, we didn’t have any tests for it previously, so it was implemented as a Stage. However, this is not accurate. You can check it here:
https://github.com/prebid/prebid-server-java/pull/1345